packages/core/admin/admin/src/content-manager/pages/ListConfiguration/components/Settings.tsx

```diff
@@ -0,0 +1,210 @@
+import React from 'react';
+
+import { Flex, Grid, GridItem, Typography } from '@strapi/design-system';
+import { useCollator } from '@strapi/helper-plugin';
+import { MessageDescriptor, useIntl } from 'react-intl';
+
+import { useEnterprise } from '../../../../hooks/useEnterprise';
+import { useForm, type InputProps } from '../../../components/Form';
+import { InputRenderer } from '../../../components/FormInputs/Renderer';
+import { useDoc } from '../../../hooks/useDocument';
+import { type EditFieldLayout } from '../../../hooks/useDocumentLayout';
+import { getTranslation } from '../../../utils/translations';
+import { type FormData } from '../ListConfigurationPage';
+
+const EXCLUDED_SORT_ATTRIBUTE_TYPES = [
+  'media',
```

Comment:
Should we try and provide this from the upload plugin at some point?

Reply:
That's a nice idea. The plugin isn't setup to export anything yet, but one to do for sure